[TOOLS-4572] improve charactor column size control function

Kang-dot opened the pull request:
http://jira.cubrid.org/browse/TOOLS-4572

- Purpose
cubrid version has been updated, it is no longer necessary to adjust the parameter type size in the latest version, so add conditions to the dialog to improve the function.

- Implementation
N/A

- Remarks
N/A

Kang-dot committed:
[TOOLS-4572] change condition for charactor type column size dialog

- add condition which target is offline(csv, dump)
- add condition which target is online(if target cubrid version is lower
  then 9.3, dialog will show)

Kang-dot committed:
Merge branch 'CUBRID:develop' into TOOLS-4572

Kang-dot committed:
[TOOLS-4527] apply code formatter

- apply code format in changed file

Kang-dot committed:
[TOOLS-4572] edit messages

- edit message property files

Srltas commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties:
```diff
@@ -484,7 +484,7 @@ msgCanceled=Canceled
 msgCfmDeleteSQL=Do you want to delete selected SQL?
 msgCfmErrorSave=Error occurred when saving configuration and some configuration may be lost.\r\nDo you want to leave this page?
-msgCheckCharset=** Notice **\n\nTo avoid miss any data unexpectedly, please confirm charactor columns' size before migration.\n\nBefore CUBRID 9.3, the length definition of a charactor column means how many bytes can be stored into the column, for example\: char(10) can store 10 single byte charactors or 5 double bytes charactors.\nSince CUBRID 9.3, the length means how many charactors can be sto ...
+msgCheckCharset=** Notice **\n\nTo avoid miss any data unexpectedly, please confirm charactor columns' size before migration.\n\Less than CUBRID 9.3, the length definition of a charactor column means how many bytes can be stored into the column, for example\: char(10) can store 10 single byte charactors or 5 double bytes charactors.\nSince CUBRID 9.3, the length means how many charactors can be s ...
```
The line break string 'n' is missing, so there is no line space.

Kang-dot replied:
i'll add missing charactor in next commit

Srltas commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/utils/MigrationCfgUtils.java:
```diff
@@ -889,8 +889,17 @@ protected VerifyResultMessages createVerifyResult(
-                || DatabaseType.ORACLE.equals(config.getSourceDBType());
+        boolean isEffectedByCharacterTypeSize;
+
+        if (config.targetIsDBDump() || config.targetIsCSV()) {
```
Changing it to `config.targetIsOnline()` is clearer.
If you use the code you wrote, you should also add SQL and XLS.

Kang-dot replied:
change if condition statement. it cover all 'target file' migration

Srltas commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/utils/MigrationCfgUtils.java:
```diff
@@ -889,8 +889,17 @@ protected VerifyResultMessages createVerifyResult(
+            isEffectedByCharacterTypeSize = !config.isAddUserSchema();
+        } else {
+            isEffectedByCharacterTypeSize =
+                    Integer.parseInt(config.getTargetDBVersion()) < 93 ? true : false;
```
There is no problem with the code working without `? true : false`.

Kang-dot replied:
i'll remove unnecessary code in next commit

Kang-dot committed:
[TOOLS-4572] edit messages

- edit messages

Kang-dot committed:
[TOOLS-4572] edit method code

- change if statement condition. it covers all 'target is file' migration
  case
- remove unnecessary ternary operator

Kang-dot committed:
[TOOLS-4572] apply code format

- apply google java format

Srltas commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/utils/MigrationCfgUtils.java:
```diff
@@ -889,8 +889,16 @@ protected VerifyResultMessages createVerifyResult(
+        if (config.targetIsFile()) {
+            isEffectedByCharacterTypeSize = !config.isAddUserSchema();
+        } else {
+            isEffectedByCharacterTypeSize = Integer.parseInt(config.getTargetDBVersion()) < 93;
```
How about making the integer '93' a constant and using it?
ex) `CUBRID_9_3_UNDER`

Kang-dot replied:
i think that is good suggestion and i agree that. but i think if we make '93' to static string variable, either '112' will apply that change. and this change need working in another issue.

Srltas replied:
I don't understand that 112 is affected by the correction.
In the view of other developers, the integer '93' is thought to be misleading, so the opinion is to explain it with the variable name.

Kang-dot replied:
i'm checking code now, some files use '112' as it is, and other some file use static variable. they are not consistent. 
so how about this? in this issue, make '93' to constant variable. and the '112' values which is not variable will change to constant variable in other issue.

Srltas replied:
In the current issue, we generate a constant for '93', and we solve the constant problem of '112' by creating another issue.

Kang-dot committed:
[TOOLS-4572] add '93' value to static string

- add '93' value to static string
